nextcord/scheduled_events.py

```diff
@@ -204,8 +204,10 @@ class ScheduledEvent(Hashable):
         The scheduled start time for the event.
     user_count: :class:`int`
         An approximate count of the 'interested' users.
+    cover: :class:`Asset`
+        The event cover image
     """
-    __slots__: Tuple[str] = (
+    __slots__: Tuple[str,...] = (
```

Comment:
Why ellipsis in type hint?

Reply:
Means there are more than one of that type.